src/LinAlg/hiopMatrixRajaSparseTriplet.cpp

```diff
@@ -1545,13 +1615,11 @@ void hiopMatrixRajaSparseTriplet::setSubmatrixToConstantDiag_w_colpattern(const
   assert(nrm == nnz_to_copy);
 #endif
 
-#if 0 // implemenation that requires a RAJA new version released in Nov.2021
+#if 1 // implemenation that requires a RAJA new version released in Nov.2021
```

Comment:
Same comment about this being `ifdef`ed out. If the impls are equivilant and we would like to keep both around, I would prefer to have a macro at the top of this source file to control the behavior, something like `HIOP_MATRIX_RAJA_SP_TRIPLET_USE_RAJA_INCLUSIVE_SCAN` (naming is hard).